templates/taxi/driver_detail.html

```diff
@@ -22,5 +23,6 @@
     {% empty %}
       <p>No cars!</p>
     {% endfor %}
+  <a href="{% url 'taxi:driver-delete' driver.id %}"><button class="btn btn-primary">Delete</button></a>
```

Comment:
Ensure that the `url` tag syntax is correct when passing arguments. It should be `{% url 'taxi:driver-delete' driver.id %}`.